### PR TITLE
MultiPartUpload.get_all_parts return 1,000 multipart uploads max

### PR DESCRIPTION
**Pull request by islue**

### CODE REVIEW

File: mongodb_consistent_backup/Upload/S3/S3.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import os
 import logging
 
+import boto
```
Comment (timvaillancourt) [inline mongodb_consistent_backup/Upload/S3/S3.py]: Is it possible to import less of boto here?

Example:

```
import boto.s3.multipart
```
?

Comment (timvaillancourt) [inline mongodb_consistent_backup/Upload/S3/S3.py]: Also let's put it on line #1 so it's in alphabetical order, if possible.

Comment (rob256) [inline mongodb_consistent_backup/Upload/S3/S3.py]: PEP8 says to put imports from the standard library at the top, and imports from third party packages underneath as islue has done.

Comment (timvaillancourt) [inline mongodb_consistent_backup/Upload/S3/S3.py]: Makes sense @rob256. My only suggestion is to import boto.s3.multipart only, then.

**Comment by islue** [comment]
My pleasure to fix the import issue.
I intended to use force push to merge the tiny fix but also removed reviews. Sorry for that, I'll be careful next time.

**Comment by timvaillancourt** [comment]
Thanks @islue, this looks good!

**Pull request merged**